### PR TITLE
Compatfix/collection self

### PR DESCRIPTION
**Pull request by joshco**
bug: osdi service was not returning self links on collections, nor top level curies.  This also includes a better fix for returning 200 on empty collections based on @shaisachs factoring concerns in #7

### CODE REVIEW

File: lib/bridge.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function getVANPaginationParams(req) {
     top = pagination.perPage;
   }
 
-  return { 'top': top, 'skip': skip };
+  return { 'top': top, 'skip': skip, url: req.url };
```
Comment (itsdrewmiller) [inline lib/bridge.js]: probably should also be quoted for the property name here, although I imagine it still works without it.

**Comment by itsdrewmiller** [comment]
Definitely thought you were misspelling "query" until I looked at the code.  What is a curie?

**Comment by itsdrewmiller** [comment]
(If you can fix the url thing the rest looks good.)

**Comment by joshco** [comment]
Curie is a hypermedia thang, it's the mechanism for prefixing like osdi:people or van:shift.  The curie definitions identify the prefix and the URL for documentation on what it means.  These docs links are exposed in the HAL browser too.
you'll see that at https://vangogh.splitatom.com

**Pull request merged**